<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeeSavedArraysWindow</class>
 <widget class="QDialog" name="SeeSavedArraysWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Сохраненные массивы</string>
  </property>
  <widget class="QPushButton" name="UpdateArrayButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Обновить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NumberOfArrayElementsLineEdit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>90</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PlusButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>128</y>
     <width>41</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QTableView" name="ArraysTableview">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>341</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteArrayButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MinusButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>128</y>
     <width>41</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SeeArrayButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Просмотреть</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LoadArrayButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Загрузить</string>
   </property>
  </widget>
  <widget class="QListView" name="CurrentArrayListview">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>121</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
